src/ApiService/ApiService/onefuzzlib/ScalesetOperations.cs

```diff
@@ -3,15 +3,24 @@
 
 namespace Microsoft.OneFuzz.Service;
 
-public interface IScalesetOperations : IOrm<Scaleset> {
+public interface IScalesetOperations : IStatefulOrm<Scaleset, ScalesetState> {
     IAsyncEnumerable<Scaleset> Search();
 
-    public IAsyncEnumerable<Scaleset> SearchByPool(PoolName poolName);
+    IAsyncEnumerable<Scaleset?> SearchByPool(PoolName poolName);
```

Comment:
Could the nullable modifier on `Scaleset` be removed since the implementation is returning non-nullable?